d-rita/fast-food-fast-react
# #164353009 Set up sass processing client

**What does this PR do?**
This PR sets up sass and CSS processing client for the project. 

**Description of tasks completed**
- Install sass-loader, node-sass
- Add scss to project
- Configure webpack to use sccs and css

**How this should be manually tested**
- Fetch and check out into this branch using `git fetch origin ch-set-up-sass-164353009 && git checkout ch-set-up-sass-164353009`
- Install dependencies with `yarn install`
- Start server using `yarn start` and go to `localhost:8080/home` to see the visual styles implemented.

**Related Pivotal Tracker Story** 
[#164353009](https://www.pivotaltracker.com/story/show/164353009)
